tasks/installation.yml

```diff
@@ -10,6 +20,22 @@
   retries: 10
   delay: 2
 
+- name: check if icingadb table and user in mysql exist
+  shell: >
+    set -o pipefail;
+    mysql
+    --user={{ icinga2_master_db_root_user }}
+    --password={{ icinga2_master_db_root_pass }}
+    --host={{ icinga2_master_icinga2_database_host }}
+    --ssl-ca={{ icinga2_master_ca_path }}
+    {{ icinga2_master_icinga2_database_name }}
+    -e "SHOW TABLES;" | grep "icinga_hosts"
+  register: icinga2_master_register_icinga2_imported
+  changed_when: icinga2_master_register_icinga2_imported.rc == 1
+  failed_when: "'Access denied for' in icinga2_master_register_icinga2_imported.stderr"
+  when: icinga2_master_icingadb_enabled|bool
+  check_mode: no
```

Comment:
* Dont use "yes". Use "true" or "false"
* the "-e" flag is for grep, mysql has no "-e" flag.